src/store/reducers/index.js

```diff
@@ -108,6 +108,10 @@ export const reducer = (
 			blocks.splice( index, 1 );
 			return { blocks: blocks, refresh: ! state.refresh };
 		}
+		case ActionTypes.BLOCK.PARSE: {
+			const parsed = action.parse(action.payload)
+			return { blocks: parsed, refresh: state.refresh };
```

Comment:
The whole idea of using Redux store is to take advantage of immutability. when you update reference to the `blocks`, you can treat it as an indicator if something has changed. This makes usage of `refresh` boolean flag obsolete as it tracks the state which can be derived from `blocks`.

Reply:
So, we should remove `refresh` from the store, right?
In that case I'd prefer to do that on a separate PR.

@hypest or @mzorz might know why it was necessary to add it?